src/main/java/io/netty/buffer/api/SizeClassedMemoryPool.java

```diff
@@ -86,12 +86,16 @@ public void close() {
 
     @Override
     public void drop(Buf buf) {
+        if (closed) {
```

Comment:
@chrisvest we just do the "extra check" here to shortcut the "closed" case and make it cheaper, correct ?

Reply:
Not quite, in order to dispose of a buffer, can't put it into the pool first, because when it comes back out, it would get a new Buf instance. So we need to specifically dispose *this* Buf instance.

Reply:
got it... thanks